src/data/services/lms/api.js

```diff
@@ -54,6 +54,17 @@ const fetchSubmission = (submissionUUID) => get(
   }),
 ).then(response => response.data);
 
+/**
+ * get(':courseId/api/v1/viewer-url/:submissionUUID')
+ * @return {
+ *  url: <string>
+ *  file_name: <string>
+ * }
+ */
+const fetchTurnitinViewers = (submissionUUID, courseId) => get(
+  stringifyUrl(`${urls.fetchTurnitinViewersUrl()}/${courseId}/api/v1/viewer-url/${submissionUUID}/`)
```

Comment:
We'll need to change the API so it accepts query params instead if we wanna follow the same patterns.